spk/plexpy-custom/src/config.ini

```diff
@@ -0,0 +1,3 @@
+[General]
+launch_browser = 0
+web_port = 8183
```

Comment:
Yep, older packages don't necessarily conform to that convention. Not much we can do about it without introducing issues, but we do want to enforce it for new packages.